pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>11</java.version>
+        <java.version>21</java.version>
```

Comment:
Something else: I see that this project is still using an old way of configuring the Java source/target version. You could remove the `java.version` property (and the two references to it in the maven-compiler-plugin) and add a new property:
```
<maven.compiler.release>11</maven.compiler.release>
```
which is what we do in other models (CDM/DRR). This property is automatically picked up.